data/gegno/gegno missions.txt

```diff
@@ -0,0 +1,103 @@
+			`	One of the Quarg finally turns towards you and motions you to remain where you are for a little longer, offering you a seat. Their look of confusion disappears when, what sounded like a very faint, low-toned sigh, comes from them focusing on you again. Meanwhile, the other Quarg that was sat down has left the room, along with the ones that first brought you here.`
+			choice
+				`	"Who are these aliens I've been seeing?"`
+				`	"Is it a problem that I'm here?"`
+				`	Do not say anything.`
+			`	The Quarg continues to look at you and bows its head slightly in an apologetic manner, as it seems they, or any other Quarg on this ringworld, do not speak your language. You continue to patiently wait in the uneasy silence, withholding your curiosities as almost an hour goes by.`
+			`	Finally, the silence is broken when the second Quarg from earlier enters the room again. This time, they glare at you much more energetically, and you begin to wonder if they are going to escort you off of the ring for being somewhere you shouldn't be.`
+			`	You preemptively begin to stand up, but your action is caught short when you see the second Quarg raise its arms outward in a gesture you never thought you'd see from a Quarg. "Powerful day, young traverser, this must be for you. A very substantial first this is for both of us, I strongly imagine." It is now obvious that this Quarg is not the same one from earlier, and you begin to see a few inconsistencies in their attire and figure.`
+			choice
+				`	"Where am I?"`
+			`	"You, brave rover, have found yourself in the dwelling of the mighty Gegno, who have rooted their hardy empire here in the great galactic east." You wonder if the Quarg's use of boastful adjectives is because of gaps in his translations, or if they are intentionally speaking of the Gegno that way. Either way, they leave an odd impression on you, coming from the lower tones of Quarg speech, rather than something more excitable.`
+			choice
+				`	"The Gegno?"`
+					goto gegno
+				`	"Are the ships I’ve seen these Gegno?"`
+					goto ships
+			label gegno
+			`	"Yes, the imperious Gegno. A juvenile race, they are, many decades younger than that of you humans. They do not find themselves in the struggle of leadership the same way as others do. Rather, they glorify their pursuit of guidance between one another to share a noble future."`
+			choice 
+				`	"What is this guidance they pursue?"`
+					goto guidance
+			label ships
+			`	"Indeed, the vessels of such stature represent the robust Gegno. Not in sorrow, but greatness do they pursue their future, bearing the fruit of sacrifice as a trophy to their name." The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration."`
+			choice
+				`	"Why do they pursue in the name of sacrifice?"`
```

Comment:
The previous line did not suggest to me to ask this question. I'm not even sure this question makes sense, either in general or in relation to the previous line.

Reply:
Would it be better for the player to ask what has made the Quarg frustrated? That may be too much of an assumption, but I'm thinking about what can lead them to the next bit.